src/SonarScanner.MSBuild.PreProcessor/TeamBuildPreProcessor.cs

```diff
@@ -157,6 +157,9 @@ private async Task<bool> DoExecute(ProcessedArgs settings)
                 logger.LogDebug(Resources.MSG_Processing_PullRequest_NoBranch);
             }
 
+            var cacheProcessor = new CacheProcessor(server, settings, logger);
+            await cacheProcessor.Execute();
```

Comment:
```suggestion
            await new CacheProcessor(server, settings, logger).Execute();
```

we don't need a variable, it can be executed directly.